spec/03-plugins/17-jwt/03-access_spec.lua

```diff
@@ -277,6 +277,20 @@ describe("Plugin: jwt (access)", function()
       })
       assert.res_status(200, res)
     end)
+    it("finds the JWT if given in cookie", function()
+      PAYLOAD.iss = jwt_secret.key
+      local jwt = jwt_encoder.encode(PAYLOAD, jwt_secret.secret)
+      local authorization = "Bearer " .. jwt
+      local res = assert(proxy_client:send {
+        method  = "GET",
+        path    = "/request",
+        headers = {
+          ["Host"] = "jwt1.com",
+          ["Set-Cookie"] = "Authorization=" .. authorization .. "; path=/;domain=.jwt1.com"
+        }
+      })
+      assert.res_status(200, res)
+    end)
```

Comment:
I think we need another test case, an illegal formatted JWT token, to ensure is returns a proper `unauthorized` instead of a `500 internal server error`

Reply:
add it

Reply:
@Tieske it looks like @icyxp complied with your request. LGTM.